tests/store/crud/read/test_read_analysis.py

```diff
@@ -15,63 +15,63 @@
 def test_get_latest_nipt_analysis_to_upload(
     store_with_analyses_for_cases_not_uploaded_fluffy: Store,
     timestamp_now: datetime,
-    pipeline: str = Workflow.FLUFFY,
+    workflow: str = Workflow.FLUFFY,
 ):
     """Test get the latest NIPT analysis to upload."""
     # GIVEN an analysis that is not delivery reported but there exists a newer analysis
 
     # WHEN fetching the latest analysis to upload to nipt
     analyses: list[Analysis] = (
-        store_with_analyses_for_cases_not_uploaded_fluffy.get_latest_analysis_to_upload_for_pipeline(
-            pipeline=pipeline
+        store_with_analyses_for_cases_not_uploaded_fluffy.get_latest_analysis_to_upload_for_workflow(
+            workflow=workflow
         )
     )
 
     # THEN only the newest analysis should be returned
     for analysis in analyses:
         assert analysis.started_at == timestamp_now
         assert analysis.uploaded_at is None
-        assert analysis.pipeline == pipeline
+        assert analysis.pipeline == workflow
```

Comment:
Has this changed in the model already?
```suggestion
        assert analysis.workflow == workflow
```